front/app/modules/commercial/project_folders/admin/containers/index.tsx

```diff
@@ -129,7 +129,7 @@ const Data = adopt<DataProps, InputProps & WithRouterProps>({
   ),
 });
 
-export default withRouter((inputProps: InputProps & WithRouterProps) => (
+export default withRouter((inputProps) => (
```

Comment:
would still lieave the InputProps typing here. If there are no inputProps, just remove it, but no untyped props please. :-)